tutorial/settings.py

```diff
@@ -122,7 +122,7 @@
 # Static files (CSS, JavaScript, Images)
 # https://docs.djangoproject.com/en/1.11/howto/static-files/
 
-STATIC_URL = '/' + os.getenv('FUNCTIONS_MOUNT_POINT') + '/static/'
+STATIC_URL = '/' + os.getenv('FUNCTIONS_MOUNT_POINT', 'api/serverless') + '/static/'
```

Comment:
Ah, yes: `FUNCTIONS_MOUNT_POINT`: needs documenting as part of the WSGI wrapper — but it should be an environment variable set exactly to `api/serverless` here. 

(Looking at seeing if I can pull it from the functions runtime somehow, rather than needing to set it explicitly.)

Reply:
This was AFAIK only an issue when trying to run python manage.py migrate, not when running under the functions host.